diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/InlineClassesRule.kt

```diff
@@ -72,6 +74,7 @@ class InlineClassesRule(private val configRule: List<RulesConfig>) : Rule("inlin
                 ?: false
 
     companion object {
+        val ktVersion = KotlinVersion(1, 4, 10)
```

Comment:
Inline classes were introduced [in 1.3](https://kotlinlang.org/docs/reference/inline-classes.html)